### ISSUE TITLE
Finetuning accuracy much lower than mentioned

### ISSUE THREAD

**Issue by atique81** [BVLC/caffe#1785]
Hi all,

I have fine-tuned the Caffe provided ILSVRC CNN following the flick fine-tuning example on 2000 flickr images. Then using Caffe, I have extracted fc7 features from these 2000 images using the original ISLVRC net and the fine-tuned net. Then I have applied a linear SVM on these two types of features (one extracted using original model, one extracted with fine-tuned model) to train two SVM models. Using these two models, I have used SVM to classify 1000 flickr test images twice (once using features extracted from Caffe using the original CNN, and once using features extracted using the fine-tuned CNN). According to theory, fine-tuned CNN extracted features should give higher classification accuracy than the original CNN. But, in my case -

1. Original CNN  accuracy was 28.30%, while fine-tuned CNN classification accuracy was 27.70%
2. During fine-tuning, my test net accuracy was only 28% when it finished all 100,000 iterations. But, according to fine-tuning example, it should be about 39%.

Could anyone please let me know what is going wrong with me?

**Comment by shelhamer** [comment]
Ask questions about examples on [caffe-users](https://groups.google.com/forum/#!forum/caffe-users) mailing list -- not on issues.

From the example:

> We also only have **80,000** images to train on, so we’d like to start with the parameters learned on the 1,000,000 ImageNet images, and fine-tune as needed.
> [...]
> Once the model is fully fine-tuned on the **whole training set** over 100,000 iterations the final validation accuracy is 39.16%.

You didn't fine-tune on the whole Flickr training set so the model is overfitting.
